src/satellite_images_nso_datascience/annotations/data_preparation.py

```diff
@@ -68,8 +73,8 @@ def fill_pixel_columns(
 def extract_dataframe_pixels_values_from_tif_and_polygons(
     tif_dataset: rasterio.DatasetReader,
     polygon_gdf: gpd.GeoDataFrame,
-    name_tif_file: str,
-    name_table: str,
+    name_tif_file=False,
```

Comment:
This is very confusing. The name_tif_file and name_table variables should clearly be strings. But now you make them "False" (a boolean) by default. Why would you do that?

Reply:
this used if you there are annotations from multiple .tif files in one file.
But not all files with annotations have multiple annotations from diffferent .tif files

Reply:
I understand what it does, it just isn't a very good way to do this. You could default it to None instead of False and it should work similarly.

Reply:
I have changed this.